src/core/controller/post/postController.js

```diff
@@ -35,9 +35,13 @@ const getPost = async (request, response) => {
         response
       );
     }
-    let postData = await postModel.getPostComments(id);
+    const commentData = await postModel.getPostComments(id);
+    const likeData = await postModel.getPostLikes(id);
+    let postData = commentData.concat(likeData);
```

Comment:
can be renamed as comments, likes and posts. postData doesn't sounds good, as each and everything is a data, and we cannot add data as a suffix everywhere.